XCGLogger/Library/XCGLogger/XCGLogWrapper.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  XCGLogWrapper.swift
+//  XCGLogger
+//
+//  Created by Anders Borch on 3/7/16.
+//  Copyright © 2016 Cerebral Gardens. All rights reserved.
+//
+
+import Foundation
+
+@objc public enum LogLevel: Int {
```

Comment:
The other LogLevel struct is already hidden from objective-c by virtue of being objective-c incompatible. This log level is ignored by Swift by virtue of strict typing.

We can definitely rename one struct or the other to avoid ambiguity and improve readability, but the current state is that this actually works as is.